.github/workflows/auto-review-and-pr-summary.yml

```
@@ -0,0 +1,90 @@
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+
+      - name: Generate PR body using LLM
+        id: generate_pr_body
+        env:
+          SUMMARY: ${{ steps.wait_for_comment.outputs.summary }}
+          GEMINI_API_KEY: ${{ secrets.GEMINI_API_KEY }}
+        run: |
+          # PRテンプレートを読み込む
+          PR_TEMPLATE=$(cat .github/PULL_REQUEST_TEMPLATE.md)
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)